src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -114,11 +121,9 @@
         allow_trailing=True,
     ),
     QuestionMarkSegment=StringParser("?", SymbolSegment, type="question_mark"),
-    AtSignLiteralSegment=NamedParser(
+    AtSignLiteralSegment=TypedParser(
         "at_sign_literal",
         ansi.LiteralSegment,
-        type="at_sign_literal",
-        trim_chars=("@",),
```

Comment:
I'm curious -- what was `trim_chars` doing, and how do we no longer need it?

Reply:
Ah - we still need it - it's just defined further up now. Given the segment in the lexer is of type `at_sign_literal` and of the class `LiteralSegment` - it won't mutate and these values wouldn't be applied anyway if set.